cmd/minishift/cmd/start_preflight_notwin.go

```diff
@@ -0,0 +1,25 @@
+// +build !windows
```

Comment:
This is not intended I think. Can you please remove it?

Reply:
This is needed, to prevent it from getting compiled in case of building for windows.

Reply:
It is needed. There are two OS specific versions of _checkVBoxInstalled_